### PR TITLE
chore: extract converters

### PR DESCRIPTION
**Pull request by octonato**
PassivationStragety and Metadata conversions extracted to some utility objects. We need that for all entity adapters. 

Kind of boring to always convert that again and again.

### CODE REVIEW

File: sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/MetadataConverters.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk.impl
+
+import com.akkaserverless.javasdk
+import com.akkaserverless.scalasdk
+
+object MetadataConverters {
+  def toScala(javaSdkMetadata: javasdk.Metadata): scalasdk.Metadata =
```
Comment (RayRoestenburg) [inline sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/MetadataConverters.scala:23]: Maybe nicer to add an apply to `scalasdk.Metadata` object?

**Review by RayRoestenburg — APPROVED** [review]
LGTM

File: sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/MetadataConverters.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk.impl
+
+import com.akkaserverless.javasdk
+import com.akkaserverless.scalasdk
+
+object MetadataConverters {
```
Comment (patriknw) [inline sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/impl/MetadataConverters.scala:22]:
```suggestion
private[scalasdk] object MetadataConverters {
```

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
Ups, I missed that #540 is on top of this and merged it already, sorry

**Pull request merged**